pgsqltoolsservice/connection/connection_service.py

```diff
@@ -24,10 +22,12 @@
     GET_CONNECTION_STRING_REQUEST, GetConnectionStringParams,
     LIST_DATABASES_REQUEST, ListDatabasesParams, ListDatabasesResponse
 )
+
 from pgsqltoolsservice.hosting import RequestContext, ServiceProvider
 from pgsqltoolsservice.utils import constants
 from pgsqltoolsservice.utils.cancellation import CancellationToken
-
+from pgsqltoolsservice.driver import *
+from pgsqltoolsservice.parsers.owner_uri_parser import get_attribute_value
 
 class ConnectionInfo(object):
```

Comment:
How does ServerConnection related to ConnectionInfo ?

Reply:
ConnectionInfo contains mappings between URI's and connections. The only difference from before is that the connections are of class ServerConnection, instead of psycopg connections as before.